MuxExoPlayer/src/r2_11_1/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java

```diff
@@ -401,7 +399,11 @@ public void onPlayerError(ExoPlaybackException e) {
 
     @Override
     public void onPositionDiscontinuity(int reason) {
-
+        if (reason == Player.DISCONTINUITY_REASON_SEEK) {
+            if ( state == PlayerState.PAUSED || !playItemHaveVideoTrack ) {
+                seeked(false);
+            }
+        }
     }
```

Comment:
See if it's possible to move this method implementation to `MuxBaseExoPlayer` to avoid repetition across different version of ExoPlayer.

Reply:
This function is inherited from Player.EventListener if we move it to MuxBaseExoPlayer we would have to move all the code implemented in that interface to MuxBaseExoPlayer which is not a good idea I think.

I could implement a function in MuxBaseExoPlayer that will do the same logic and have it called from this function.